perf_testing/scripts/highspeed_write_nonzero.py

```diff
@@ -0,0 +1,77 @@
+import os
+import shutil
+import time
+import argparse
+import subprocess
+import mmap
+import io
+from multiprocessing import Pool, cpu_count
+
+
+# Function to create files using dd command
+def create_file_dd(file_index, folder, source_file, timestamp):
```

Comment:
Are we using this script in the pipeline anywhere or its just for testing manually for high speed writes.